.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Please remove this. It is much easier to create [global gitignore](https://www.google.com/search?q=global+.gitignore) once.